node-definitions/sonic/SONiCv.yaml

```diff
@@ -0,0 +1,61 @@
+id: SONiCv
+general:
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - Eth0
+      - Ethernet0
+      - Ethernet4
+      - Ethernet8
+      - Ethernet12
+      - Ethernet16
+      - Ethernet20
+      - Ethernet24
+      - Ethernet28
+    serial_ports: 2
+    default_count: 9
+    loopback:
+      - Loopback0
+ui:
+  visible: true
+  label_prefix: SONiCv-
+  icon: switch
+  label: SONiC Virtual Switch (VM)
+sim:
+  linux_native:
+    libvirt_domain_driver: kvm
+    driver: server
+    disk_driver: virtio
+    ram: 4096
+    cpus: 2
+    cpu_limit: 25
+    nic_driver: e1000
+    video:
```

Comment:
Does it need a video console in addition to serial?

Reply:
Probably copy paste error, it does not require video will remove it.